app/src/main/res/values-fr/strings.xml

```diff
@@ -243,11 +243,11 @@
     <string name="theme_amoled">Sombre AMOLED</string>
     <string name="theme_semi_dark">Semi-transparent sombre</string>
     <string name="theme_semi_light">Semi-transparent</string>
-    <string name="double_tap_to_lock">Double-tap pour vérouiller l\'écran</string>
-    <string name="enable_double_tap_to_lock">Pour activer le verrouillage par double tap, vous devrez accorder des permissions supplémentaires : activez KISS dans l\'écran suivant.</string>
+    <string name="double_tap_to_lock">Appui double pour verrouiller l\'écran</string>
+    <string name="enable_double_tap_to_lock">Pour activer le verrouillage par double appui, vous devrez accorder des permissions supplémentaires : autorisez KISS dans l\'écran suivant.</string>
```

Comment:
"autorisez" sounds more accurate to me, but that's not the hill I want to die on, so let me know :)